google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITDownloadToTest.java

```diff
@@ -19,51 +19,102 @@
+import com.google.cloud.storage.conformance.retry.ParallelParameterized;
+import com.google.cloud.storage.conformance.retry.TestBench;
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Arrays;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized.Parameters;
 
+@RunWith(ParallelParameterized.class)
 public final class ITDownloadToTest {
+  @ClassRule(order = 0)
+  public static final TestBench TEST_BENCH =
+      TestBench.newBuilder().setContainerName("it-grpc").build();
 
   @ClassRule(order = 1)
-  public static final StorageFixture storageFixture = StorageFixture.defaultHttp();
+  public static final StorageFixture storageFixtureGrpc =
+      StorageFixture.from(
+          () ->
+              StorageOptions.grpc()
+                  .setHost(TEST_BENCH.getGRPCBaseUri())
+                  .setCredentials(NoCredentials.getInstance())
+                  .setProjectId("test-project-id")
+                  .build());
+
+  @ClassRule(order = 1)
+  public static final StorageFixture storageFixtureHttp = StorageFixture.defaultHttp();
+
+  @ClassRule(order = 2)
+  public static final BucketFixture bucketFixtureGrpc =
+      BucketFixture.newBuilder()
+          .setBucketNameFmtString("java-storage-grpc-%s")
+          .setCleanupStrategy(CleanupStrategy.ALWAYS)
+          .setHandle(storageFixtureGrpc::getInstance)
+          .build();
 
   @ClassRule(order = 2)
-  public static final BucketFixture bucketFixture =
-      BucketFixture.newBuilder().setHandle(storageFixture::getInstance).build();
+  public static final BucketFixture bucketFixtureHttp =
+      BucketFixture.newBuilder()
+          .setBucketNameFmtString("java-storage-http-%s")
+          .setCleanupStrategy(CleanupStrategy.ALWAYS)
+          .setHandle(storageFixtureHttp::getInstance)
+          .build();
 
   private static final byte[] helloWorldTextBytes = "hello world".getBytes();
   private static final byte[] helloWorldGzipBytes = TestUtils.gzipBytes(helloWorldTextBytes);
 
-  private static Storage storage;
-  private static BlobId blobId;
+  private final StorageFixture storageFixture;
+  private final BucketFixture bucketFixture;
+  private final String clientName;
 
-  @BeforeClass
-  public static void beforeClass() {
-    blobId = BlobId.of(bucketFixture.getBucketInfo().getName(), "zipped_blob");
+  public ITDownloadToTest(
+      String clientName, StorageFixture storageFixture, BucketFixture bucketFixture) {
+    this.storageFixture = storageFixture;
+    this.bucketFixture = bucketFixture;
+    this.clientName = clientName;
+  }
+
+  @Parameters(name = "{0}")
+  public static Iterable<Object[]> data() {
+    return Arrays.asList(
+        new Object[] {"JSON/storage.googleapis.com", storageFixtureHttp, bucketFixtureHttp},
```

Comment:
For data(): wondering if you can just split up "JSON/" and "GRPC/" to be its own element.

I may have missed how parameterization works with data(). Could it be a defined type that holds these values so it doesn't need to rely on generic `Object`?